contracts/access_control/lib.rs

```diff
@@ -16,7 +16,7 @@ mod access_control {
     };
     use scale::{Decode, Encode};
 
-    // address placeholder, set in the bytecode
+    // address placeholder, to be set in the bytecode
```

Comment:
Nit: `privileges` on line 90.

Reply:
Line 43 as well.

Reply:
Hmm I found just one typo, could you comment on the line?

Reply:
https://github.com/Cardinal-Cryptography/aleph-node/blob/2cecffa7689e9a05de385d17da0a4af03f674733/contracts/access_control/lib.rs#L43

Reply:
And later uses. I do not see the second typo as well, so must have mixed up files. Will find it in a sec.

Reply:
It crops up a couple of times in 3 files `contracts/button/lib.rs` (https://github.com/Cardinal-Cryptography/aleph-node/blob/2cecffa7689e9a05de385d17da0a4af03f674733/contracts/button/lib.rs#L42), `contracts/access_control/lib.rs` (these are the ones you have probably already fixed) and `contracts/scripts/deploy.sh` (in comments).

Reply:
ok, should be fixed now

Reply:
`deploy.sh` still has a couple in comments, upper case. Other than that, fixed.